docs/source/tutorials_source/platform/tutorial_active_learning_detectron2.py

```diff
@@ -0,0 +1,339 @@
+from lightly.active_learning.utils.object_detection_output import ObjectDetectionOutput
+from lightly.active_learning.scorers import ScorerObjectDetection
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.active_learning.agents import ActiveLearningAgent
+from lightly.active_learning.config import SamplerConfig
+from lightly.openapi_generated.swagger_client import SamplingMethod
+
+# %%
+# Upload dataset to Lightly
+# -------------------------
+#
+# To work with the Lightly Platform and use the active learning feature we
+# need to upload the dataset. 
+# 
+# First, head over to `the Lightly Platform <https://app.lightly.ai/>`_ and 
+# create a new dataset.
+#
+# We can now upload the data using the command line interface. Replace 
+# **yourToken** and *yourDatasetId** with the two provided values from the web app.
+# Don't forget to adjust the **input_dir** to the location of your dataset.
+#
+# .. code:: 
+# 
+#     lightly-magic token="yourToken" dataset_id="yourDatasetId" \
+#         input_dir='/datasets/comma10k/imgs/' trainer.max_epochs=20 \
+#         loader.batch_size=64 loader.num_workers=8
+#
+# .. note::
+#
+#     In this tutorial, we use the lightly-magic command which trains a model
+#     before embedding and uploading it to the Lightly Platform.
+#     To skip training, you can set `trainer.max_epochs=0`.
+
+
+YOUR_TOKEN = "yourToken"  # your token of the web platform
+YOUR_DATASET_ID = "yourDatasetId"  # the id of your dataset on the web platform
+DATASET_ROOT = '/datasets/comma10k/imgs/'
+
+# allow setting of token and dataset_id from environment variables
+def try_get_token_and_id_from_env():
+    token = os.getenv('TOKEN', YOUR_TOKEN)
+    dataset_id = os.getenv('AL_TUTORIAL_DATASET_ID', YOUR_DATASET_ID)
+    return token, dataset_id
+
+YOUR_TOKEN, YOUR_DATASET_ID = try_get_token_and_id_from_env()
+
+
+
+# %%
+# Inference on unlabeled data
+# ----------------------------
+#
+# In active learning, we want to pick the new data for which our model struggles 
+# the most. If we have an image with a single car in it and our model has 
+# high confidence that there is a car we don't gain a lot by including 
+# this example in our training data. However, if we focus on images where the 
+# model is not sure whether the object is a car or a building we want 
+# to include these images to refine the decision boundary.
+#
+# First, we need to create an active learning agent in order to 
+# provide lightly with the model predictions. 
+# We can use the ApiWorkflowClient for this. Make sure that we use the 
+# right dataset_id and token.
+
+# create Lightly API client
+api_client = ApiWorkflowClient(dataset_id=YOUR_DATASET_ID, token=YOUR_TOKEN)
+al_agent = ActiveLearningAgent(api_client)
+
+# %%
+
+# we can access the images of the dataset we want to use for active learning using
+# the `al_agent.unlabeled_set` property
+
+# let's print the first 3 entries
+print(al_agent.unlabeled_set[:3])
```

Comment:
You still use the `unlabeled_set` here. Especially the sentence "The `unlabeled_set` is the set of images from which we want to query" below (line 176) is problematic. Why not always use `query_set`?

Reply:
I recommend doing a STRG-F for "unlabeled" and replace it with `query_set` nearly everywhere.

Reply:
I'll replace it with `query_set` and let's discuss this offline in a call. I switched it to unlabeled_set because if you re-run the code twice because you did two iterations you will have to deal with the preselected tag.